!GameTexts/NewTexts.xml

```diff
@@ -409,24 +401,24 @@
 			<Text></Text>
```

Comment:
Maybe because they are specific civ-tied resources we should write something about it?

Reply:
No other resources have help text, and neither resource has any unique mechanics that would necessitate help text. Adding descriptions would actually add confusion